src/main/kotlin/net/irisshaders/lilybot/extensions/util/ModUtilities.kt

```diff
@@ -198,6 +363,44 @@ class ModUtilities : Extension() {
 		}
 	}
 
+	inner class SayEditArgs : Arguments() {
+		/** The ID of the embed to edit. */
+		val messageToEdit by snowflake {
+			name = "messageToEdit"
+			description = "The ID of the embed you'd like to edit"
+		}
+
+		/** The new content of the embed. */
+		val newContent by optionalString {
+			name = "newContent"
+			description = "The new content of the embed"
+
+			mutate {
+				it?.replace("\\n", "\n")
+					?.replace("\n", "\n")
+			}
+		}
+
+		/** The new color for the embed. */
+		val newColor by optionalColour {
+			name = "newColor"
+			description = "The new color of the embed"
+		}
+
+		/** The channel the embed was originally sent in. */
+		val channelOfMessage by optionalChannel {
+			name = "channelOfEmbed"
+			description = "The channel of the embed"
+		}
+
+		/** Whether to add the timestamp of when the message was originally sent or not. */
+		val timestamp by defaultingBoolean {
+			name = "timestamp"
+			description = "Whether to add the timestamp of when the message was originally sent or not"
+			defaultValue = false
```

Comment:
I think this should default to true for consistency with original /say. We should also account for the old embed's time stamp or lack thereof

Reply:
I do already account for that. I'll make the change now